### DISCUSSION TITLE
Deleting an IAM Group that has no Name

### DISCUSSION THREAD

**Discussion by Skarlso** [aws/aws-sdk-go-v2#2028] [Q&A]
Hello! Is there any kind of SDK API I can call that could remove an IAM group without a name? We have a invalid group that was created without a name, and now we can't delete it because the `DeleteGroup` input only accepts a `Name` and no ID or ARN.

**Comment by Skarlso** [comment]
Also, it's super weird, because I can see that in the JSON definition there is a field called Group but I can't see that in the struct.... what's happening :D

**Reply by jmklix** [reply]
Can you try deleting this group inside the AWS console manually?
